OracleEDQ/12.2.1.3.0/Dockerfile

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Enterprise Data Quality 12.2.1.3.0
+#
+# Base image of this dockerfile is Oracle Java 8 docker image.
+#
+# Pre-requisite to build oracle Oracle Enterprise Data Quality 12.2.1.3.0 image is to have
+# Oracle Java 8 docker image
+#
+# To build Oracle Java 8 docker image, kindly refer below link
+# https://github.com/oracle/docker-images/tree/master/OracleJava
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+#  fmw_12.2.1.3.0_edq_generic.zip
+#   Download the production distribution for Oracle Enterprise Data Quality from the Oracle Software Delivery Cloud
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Make sure that Oracle Java 8 image is available
+# Run:
+#      $ docker build -t oracle/edq:12.2.1.3.0 .
+#
+#
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.3
+
+#
+# Maintainer
+# ----------
+#MAINTAINER 
```

Comment:
Why are you not providing a `MAINTAINER`? This needs to be provided so we know who to assign any potential issues or pull requests to in future.

Reply:
Fixed, changed to 
#MAINTAINER Subhash Sutrave <subhash.sutrave@oracle.com>